firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionServiceImplTest.java

```diff
@@ -236,7 +237,7 @@ public void checkForNewRelease_authenticationFailure_signOutTester() throws Inte
     Task<AppDistributionRelease> task = firebaseAppDistribution.checkForNewRelease();
 
     awaitTaskFailure(task, AUTHENTICATION_FAILURE, "Test");
-    awaitTermination(testExecutor);
+    awaitTermination(lightweightExecutor);
```

Comment:
hm, these executors cannot be shutdown so I am not sure `awaitTermination` would work

Reply:
but looks like tests have passed, so use your judgement

Reply:
Hmm, I'm realizing that all of our `awaitTermination()` calls are really just sleeping for 100 ms because we're never shutting them down first. We need a better way to wait for executor-run tasks throughout our tests. Since these are passing right now though I'm inclined to tackle that in a followup.